econml/tests/test_orf.py

```diff
@@ -64,7 +67,8 @@ def test_continuous_treatments(self):
         # Test continuous treatments with controls
         est = ContinuousTreatmentOrthoForest(n_trees=50, min_leaf_size=10,
                                              max_splits=50, subsample_ratio=0.30, bootstrap=False, n_jobs=4,
-                                             model_T=Lasso(alpha=0.024), model_Y=Lasso(alpha=0.024),
+                                             model_T=WeightedModelWrapper(Lasso(alpha=0.024), sample_type="weighted"),
```

Comment:
Same thing as above: `model_T` and `model_Y` don't need to accept weights because the nuisance estimates in the causal tree are not weighted.